shell-completion/Makefile

```diff
@@ -1,29 +1,31 @@
+MAKEFLAGS += --jobs=$(shell nproc)
```

Comment:
This wouldn't work on BSDs

Reply:
Sure? As I recall this will throw a warning on some systems but the result will just be no parallel jobs, the basic function would still run.

Reply:
Yeah, it will moan and go on... Maybe just a little check to silence it:
```make
ifneq (, $(shell type nproc 2>/dev/null))
MAKEFLAGS += --jobs=$(shell nproc)
endif
```
Or something of this sort...

Reply:
Yup. I stuffed a default value in while I was at it.